akka-stream/src/main/scala/akka/stream/impl/StreamLayout.scala

```diff
@@ -251,7 +251,7 @@ private[akka] object StreamLayout {
          * materialized value computation of its submodules.
          */
         Atomic(this),
-        Attributes.none)
+        Attributes.name("nested"))
```

Comment:
Otherwise some nested modules have no name attribute which trickles down to `unknown-op` in actor names.